Cargo.toml

```diff
@@ -19,7 +19,7 @@ anyhow = "1.0"
 bitar = { version = "0.10", features=["rustls-tls"] }
 time = "0.3"
 dmg = "0.1.2"
-espim = "0.2"
+espim = "0.2.1"
```

Comment:
No need to change this, as soon as i push the espim update you can run `cargo update espim` to update your Cargo.lock file